content/chainguard/libraries/scanners.md

```diff
@@ -0,0 +1,228 @@
+---
+title: "Vulnerability Scanners and Chainguard Libraries"
+linktitle: "Vulnerability Scanners"
+description: "Details for using vulnerability scanners with Chainguard Libraries."
+type: "article"
+date: 2025-10-04T12:00:00+00:00
+lastmod: 2025-10-04T12:00:00+00:00
+draft: false
+tags: ["Chainguard Libraries"]
+menu:
+  docs:
+    parent: "libraries"
+weight: 006
+toc: true
+---
+
+Vulnerability scanners enable your organization and development teams to
+understand the potential security risks from libraries used within your
+applications.
+
+Chainguard Libraries provides a trusted source for libraries typically
+downloaded from public repositories.  Libraries are rebuilt from the upstream
+open source project code repository content only. This prevents malware without
+published source code and reduces almost all risk for software supply chain
+attacks. In addition, some library versions are available with backports of
```

Comment:
could we say "package" instead of "library" here? I feel like I'm confusing "libraries" and "Chainguard Libraries" already